ports/hdf5/portfile.cmake

```
@@ -1,20 +1,13 @@
 vcpkg_fail_port_install(ON_TARGET "UWP")
 
-vcpkg_download_distfile(ARCHIVE
-    URLS "https://support.hdfgroup.org/ftp/HDF5/releases/hdf5-1.10/hdf5-1.10.5/src/CMake-hdf5-1.10.5.tar.gz"
-    FILENAME "CMake-hdf5-1.10.5.tar.gz"
-    SHA512 a25ea28d7a511f9184d97b5b8cd4c6d52dcdcad2bffd670e24a1c9a6f98b03108014a853553fa2b00d4be7523128b5fd6a4454545e3b17ff8c66fea16a09e962
-)
-
-vcpkg_extract_source_archive_ex(
+vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
-    ARCHIVE ${ARCHIVE}
-    REF hdf5
-    PATCHES
-        hdf5_config.patch       
-        fix-generate.patch      # removes the build of static targets in shared builds
-        static-targets.patch    # maps the internal static tagets to the shared targets if building as a dynamic library
-        export-private.patch    # exports two additional functions in shared builds to make hl/tools/h5watch build in shared builds. 
+    REPO  live-clones/hdf5
+    REF hdf5-1_12_0
+    SHA512 d84df1ea72dc6fa038440a370e1b1ff523364474e7f214b967edc26d3191b2ef4fe1d9273c4a086a5945f1ad1ab6aa8dbcda495898e7967b2b73fd93dd5071e0
+    HEAD_REF develop
+    PATCHES hdf5_config.patch
```

```suggestion
    PATCHES 
        hdf5_config.patch
```
Could you please update this?